.travis.yml

```diff
@@ -61,31 +61,6 @@ matrix:
         - TEST="misc"
         - PROFILE="-Dhadoop.version=2.8.0"
         - CACHE_NAME=JDK8_H280_M
-    - jdk: "openjdk8"
```

Comment:
How about retaining these Travis profiles for now for the Hadoop and JDK versions?
Just bump the Scala dependency.

Reply:
You're right, of course. Undropping ...